examples/pingone-neo-getting-started/main.tf

```diff
@@ -0,0 +1,425 @@
+      "https://www.example.com/app/callback",
+    ]
+
+    mobile_app {
+      bundle_id    = "com.pingidentity.PingOneWalletSample"
+      package_name = "com.pingidentity.shocard"
+    }
+  }
+
+}
+
+# configure digital wallet application
+resource "pingone_digital_wallet_application" "digital_wallet" {
+  environment_id = pingone_environment.my_environment.id
+  application_id = pingone_application.my_native_app.id
+  name           = "PingOne Credentials Sample Digital Wallet App"
+  app_open_url   = "https://shocard.pingone.com/appopen"
+}
+
+## Create "Getting Started" Example Credential
+
+# configure credential type
+resource "pingone_credential_type" "gettingstarted_credential" {
+  environment_id = pingone_environment.my_environment.id
+  title          = "Neo Demonstration"
+  description    = "Getting Started Demo Credential"
+  card_type      = "NeoDemonstration"
+
+  # tip: you can use the pingone_credential_type datasource to obtain a credential template from an existing environment
+  # example:  
+  #
+  #   data "pingone_credential_type" "existing_credential" {
+  #           environment_id     = var.existing_environment_id
+  #           credential_type_id = var.existing_credential_type_id
+  #   }
+  #
+  #   assign the template to the new resource similar to:
+  #   card_design_template = pingone_credential.existing_credential_type.card_design_template
+  card_design_template = "<svg xmlns=\"http://www.w3.org/2000/svg\" xmlns:xlink=\"http://www.w3.org/1999/xlink\" viewBox=\"0 0 740 480\"><rect fill=\"none\" width=\"736\" height=\"476\" stroke=\"#CACED3\" stroke-width=\"3\" rx=\"10\" ry=\"10\" x=\"2\" y=\"2\"></rect><rect fill=\"$${cardColor}\" height=\"476\" rx=\"10\" ry=\"10\" width=\"736\" x=\"2\" y=\"2\"></rect><image href=\"$${backgroundImage}\" style=\"opacity:15%\" height=\"476\" rx=\"10\" ry=\"10\" width=\"736\" x=\"2\" y=\"2\"></image><image href=\"$${logoImage}\" x=\"42\" y=\"43\" height=\"90px\" width=\"90px\"></image><text fill=\"$${textColor}\" font-weight=\"450\" font-size=\"30\" x=\"160\" y=\"90\">$${cardTitle}</text><text fill=\"$${textColor}\" font-size=\"25\" font-weight=\"300\" x=\"160\" y=\"130\">$${cardSubtitle}</text><image href=\"$${fields[3].value.href}\" x=\"75\" y=\"200\" height=\"90px\" width=\"90px\"></image><text fill=\"$${textColor}\" font-weight=\"500\" font-size=\"25\" x=\"200\" y=\"220\">$${fields[1].value} $${fields[2].value}</text><text fill=\"$${textColor}\" font-weight=\"500\" font-size=\"20\" x=\"200\" y=\"250\">$${fields[0].title}: $${fields[0].value}</text></svg>"
+
+  metadata = {
+    name               = "NeoDemonstration"
+    description        = "Getting Started Demo Credential"
+    bg_opacity_percent = 30
+
+    # ensure images have content-type prefix defined and are base64 encoded
+    background_image = "data:image/png;base64,${filebase64("./images/gettingstarted_background.png")}"
+    logo_image       = "data:image/png;base64,${file("./images/gettingstarted_logo.png")}"
```

Comment:
line above is `filebase64`, does it matter which is used?

Reply:
Changed all images to binary, and consistently use `filebase64` in HCL for clarity.